index.html

```diff
@@ -50,61 +50,64 @@
                     <h1>Hey there. I'm Ivan.</h1>
                     <span>I’m a Software Developer</span>
                     <p>I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-                    <div class="headlineSocial">
-                        <a href="https://github.com/ivangonzalez224">
-                            <img src="images/gitHub.png" alt="">
-                        </a>
-                        <a href="https://www.linkedin.com/in/iv%C3%A1n-gonzalez-robles-957491275/">
-                            <img src="images/linkedin.png" alt="">
-                        </a>
-                        <a href="https://www.angellist.com/">
-                            <img src="images/angelist.png" alt="">
-                        </a>
-                        <a href="https://twitter.com/ivang2204">
-                            <img src="images/twitter.png" alt="">
-                        </a>
-                        <a href="https://medium.com/">
-                            <img src="images/medium.png" alt="">
-                        </a>
-                        
-                    </div>
+                </div>    
+                <div class="headlineSocial">
+                    <a href="https://github.com/ivangonzalez224">
+                        <img src="images/gitHub.png" alt="Git Hub Icon">
+                    </a>
+                    <a href="https://www.linkedin.com/in/iv%C3%A1n-gonzalez-robles-957491275/">
+                        <img src="images/linkedin.png" alt="Linkedin Icon">
+                    </a>
+                    <a href="https://www.angellist.com/">
+                        <img src="images/angelist.png" alt="Angel List Icon">
+                    </a>
+                    <a href="https://twitter.com/ivang2204">
+                        <img src="images/twitter.png" alt="Twitter Icon">
+                    </a>
+                    <a href="https://medium.com/">
+                        <img src="images/medium.png" alt="Medium Icon">
+                    </a>
                 </div>
-
                 <div class="headlineImage">
-                    <img src="images/illustration.png" alt="">
+                    <img src="images/illustration.png" alt="Robot Illustration">
                 </div>
-            </header>
+                <div class="headlineImageDesk">
+                    <img src="images/illustrationDesktop.png" alt="Robot Illustration">
+                </div>
+            </div>    
         </div>
     </section>
-    <div class="works">
+    <div class="works" id="works_section">
         <div class="wrapper">
             <div class="mainWorks">
                 <h1>My Recent Works</h1>
                 <div class="workslineTop"></div>
-                <div class="worksCard">
+                <div class="worksCard worksCard_1">
```

Comment:
- [ ] My Recent works and the horizontal line are on the same line on the Figma design while yours are on different line. Kindly fix that.
- [ ] I would like to suggest that, you reduce the margin between the line and the first project so as to match the figma design.

### Your Design
![your-work-title](https://github.com/ivangonzalez224/Portfolio-mobile-first/assets/26597432/f04725f5-c748-4032-8526-9d271f4c5450)

### Figma Design
![figma-work-title](https://github.com/ivangonzalez224/Portfolio-mobile-first/assets/26597432/33b1cace-3e31-402a-b2bf-7e5aaddffacb)